_posts/developers/apps/tools/2019-01-01-00_overview.md

```diff
@@ -50,7 +50,6 @@ Beginning developers may want to start with one of the interface libraries which
       See also this [cheat sheet](https://vincenttunru.gitlab.io/tripledoc/docs/cheatsheet) for a handy comparison between three commonly used linked data libraries.
 
   * <a name="resource">**Resource  Management Libraries**</a>
-     * [Node Solid WS](https://github.com/solid/node-solid-ws) - support for WebSockets
```

Comment:
I'm not up to date with this.. I suppose it was put to sleep. Is this repo to be archived or still in use?

Reply:
It's directly included in NSS and has no use beyond NSS, so its separate listing is an accident.